README.md

```diff
@@ -1,39 +1,11 @@
 # The SYCL-DNN neural network acceleration library
 
-SYCL-DNN is a library implementing various neural network algorithms written
-using the SYCL API, developed by Codeplay Software.
+# Project Description
```

Comment:
I really don't think that a single sentence summary of the project requires a header.

Reply:
Fair enough.